authority-portal-frontend/src/app/shared/common/portal-layout/title-bar/title-bar.component.html

```diff
@@ -21,12 +21,13 @@
     <button
       *ngFor="let tab of titleBarConfig.tabs"
       class="w-16 button flex items-center justify-center gap-2 bg-black text-white rounded-full px-2 py-1 text-sm hover:bg-brand-golden"
+      matTooltipClass="whitespace-pre-line"
       [ngClass]="{
         'bg-brand-golden': selectedTab === tab.view && !tab.isDisabled,
         'disabled:bg-gray-700': tab.isDisabled,
-
       }"
       [disabled]="tab.isDisabled"
+      [matTooltip]="tab.tooltip !== undefined ? tab.tooltip : ''"
```

Comment:
are you sure this works? If you're unlucky you might need `undefined` to prevent the directive opening up a small empty popup (had this before, please check)

Reply:
It does!